usrtools/xskmap_load_send/xskmap_load_send.c

```diff
@@ -0,0 +1,232 @@
+    struct option lgopts[] = {
+        {OPT_NO_COLOR, no_argument, NULL, OPT_NO_COLOR_NUM},
+        {NULL, 0, 0, 0}
+    };
+    // clang-format on
+    int opt, option_index;
+    char log_level[16] = {0};
+
+    /* Parse the input arguments. */
+    for (;;) {
+        opt = getopt_long(argc, argv, "hL:m:u:n:", lgopts, &option_index);
+        if (opt == EOF)
+            break;
+
+        switch (opt) {
+        case 'h':
+            print_usage(argv[0]);
+            return -1;
+        case 'L':
+            strlcpy(log_level, optarg, sizeof(log_level));
+            if (cne_log_set_level_str(log_level)) {
+                CNE_ERR("Invalid command option\n");
+                print_usage(argv[0]);
+                return -1;
+            }
+            break;
+        case 'm':
+            strlcpy(info.map_path, optarg, sizeof(info.map_path));
+            break;
+        case OPT_NO_COLOR_NUM:
+            tty_disable_color();
+            break;
+
+        default:
+            CNE_ERR("Invalid command option\n");
+            print_usage(argv[0]);
+            return -1;
+        }
+    }
+
+    return 0;
+}
+
+static int
+send_map_fd(int sock, int fd)
+{
+    char cmsgbuf[CMSG_SPACE(sizeof(int))];
+    struct msghdr msg;
+    struct iovec iov;
+    char value[UDS_MAX_CMD_LEN] = {0};
+
+    snprintf(value, sizeof(value), "%s,%d", UDS_FD_ACK_MSG, fd);
+
+    iov.iov_base = &value;
+    iov.iov_len  = strnlen(value, sizeof(value));
+
+    msg.msg_name       = NULL;
+    msg.msg_namelen    = 0;
+    msg.msg_iov        = &iov;
+    msg.msg_iovlen     = 1;
+    msg.msg_flags      = 0;
+    msg.msg_control    = cmsgbuf;
+    msg.msg_controllen = CMSG_LEN(sizeof(int));
+
+    struct cmsghdr *cmsg = CMSG_FIRSTHDR(&msg);
+
+    cmsg->cmsg_level = SOL_SOCKET;
+    cmsg->cmsg_type  = SCM_RIGHTS;
+    cmsg->cmsg_len   = CMSG_LEN(sizeof(int));
+
+    *(int *)CMSG_DATA(cmsg) = fd;
+    int ret                 = sendmsg(sock, &msg, 0);
+
+    if (ret < 0) {
+        CNE_ERR("Failed to send xsk_map fd with error=%s\n", strerror(errno));
+        return -errno;
+    }
+
+    return ret;
+}
+
+static int
+connect_host(uds_client_t *c, const char *cmd __cne_unused, const char *params __cne_unused)
+{
+
+    if (send(c->s, UDS_HOST_OK_MSG, sizeof(UDS_HOST_OK_MSG), 0) <= 0)
+        CNE_ERR_RET("Failed to send %s message %s\n", UDS_HOST_OK_MSG, strerror(errno));
+    else
+        CNE_DEBUG("Sent %s msg\n", UDS_HOST_OK_MSG);
+
+    return 0;
+}
+
+static int
+fin_ack(uds_client_t *c, const char *cmd __cne_unused, const char *params __cne_unused)
+{
+    if (send(c->s, UDS_FIN_ACK_MSG, sizeof(UDS_FIN_ACK_MSG), 0) <= 0)
+        CNE_ERR_RET("Failed to send %s message %s\n", UDS_FIN_ACK_MSG, strerror(errno));
+    else
+        CNE_DEBUG("Sent %s msg\n", UDS_FIN_ACK_MSG);
+
+    return 0;
+}
+
+static int
+send_xskmap_fd(uds_client_t *c, const char *cmd __cne_unused, const char *params __cne_unused)
+{
+    send_map_fd(c->s, c->info->xsk_map_fd);
```

Comment:
Need to test the return value here.

Reply:
going to forward the return value through to the calling function through the client handler